File: 01.Java-Basic/01.First-Steps-In-Coding-Lab/src/com/company/ProjectsCreation.java
package com.company;

import java.util.Scanner;

public class ProjectsCreation {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String name = scanner.nextLine();
        int projectsCount = Integer.parseInt(scanner.nextLine());
        int totalTime = projectsCount * 3;
        System.out.printf("The architect %s will need %d hours to complete %d project/s.", name, totalTime, projectsCount);
    }
}

File: JavaScript/01.JavaScriptBasic/01FirstStepsLab/ProjectsCreation.js
function creation(input) {
    let name = input[0];
    let projectsCount = Number(input[1]);
    let hours = projectsCount * 3;
    console.log("The architect " + name + " will need " + hours + " hours to complete " + projectsCount + " project/s.");
}

creation(["Kosta", "4"])